### PR TITLE
Preparing to launch a new expression of interest form for Peer Reviewers (EN, ES, FR)

### LINKED ISSUE #3323: Preparing to launch a new expression of interest form for Peer Reviewers EN, ES, FR
I am opening this Issue to prepare to launch a new expression of interest form for **Peer Reviewers**. 

This is part of on-going work in dialogue with the Managing Editors. Our shared aim is to support a more effective and more time-efficient system for recruiting and managing an active pool of volunteer peer reviewers. 

An expression-of-interest form managed by the Publishing Team will empower reviewers to sign up and express their interest in participating, and will help us to improve the experience for all.

**Note**: We are preparing this launch in EN, ES, and FR (PT to follow in dialogue with E.B.)

---

We've prepared an update consisting of three components:

> **1**. An additional, short paragraph on the /en/contribute, /es/contribuciones, and /fr/contribuer pages.

> **2**. Links to 3x Google Forms (one per language) to be managed by the Publishing Team in association with the Managing Editors where reviewers can sign up to express their interest in participating. The forms are set up to collate responses in a spreadsheet which each ME (as well as Charlotte and I) will have access to.

> **3**. 3x plain-text versions of the sign-up form which potential reviewers can send to us by email if they prefer. The Publishing Team will support the MEs by manually adding any responses received using the plain text version of the form to the appropriate shared spreadsheet(s).

---

This initiative will be supported by communications across:
- blog posts
- social media posts (Twitter/X, Mastodon, LinkedIn, Patreon) in the target languages
- announcements in our Bulletin // Boletín // Bulletin // Boletim
- community mailings `<ph-circulation-calls-opportunities.xlsx>` in the target languages

### PR DESCRIPTION
**Pull request by anisa-hawes**
We are preparing to launch a new expression of interest form for Peer Reviewers (EN, ES, FR).

**Note:** PT to follow in dialogue with E.B.

Closes #3323

--

This update consists of three components:

1. An additional, short paragraph on the /en/contribute, /es/contribuciones, and /fr/contribuer pages.

2. **Links to** 3x Google Forms (one per language) to be managed by the Publishing Team in association with the Managing Editors where reviewers can sign up to express their interest in participating. The forms are set up to collate responses in a spreadsheet which each ME (as well as Charlotte and I) will have access to.

3. 3x plain-text versions of the sign-up form which potential reviewers can send to us by email if they prefer. The Publishing Team will support the MEs by manually adding any responses received using the plain text version of the form to the appropriate shared spreadsheet(s).

--

Thanks to Charlotte and Jenn for translations into FR and ES ✨ 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

### CODE REVIEW

File: fr/contribuer.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ Nous sommes loin de simplement accepter ou rejeter des articles, comme le font l
 
 Si vous souhaitez proposer une leçon, que vous en soyez l'auteur(e) ou pas, merci [d'envoyer un courriel à notre rédacteur/rédactrice en chef](mailto:francais@programminghistorian.org).
 
+## Évaluer une leçon 
+
+Nous sommes fiers de notre politique ouverte d'évaluation par les pairs. Nous pensons que ce système offre une occasion précieuse de collaboration entre les auteurs/res, les traducteur/rices et leurs pairs de la communauté internationale d’humanités numériques. L’évaluation ouverte favorise un échange de connaissances productif et nous aide à élaborer des leçons utiles, accessibles et durables. 
+
+Si vous souhaitez contribuer en tant qu’évaluateur, veuillez prendre quelques minutes pour [faire part de votre intérêt via notre formulaire Google](https://forms.gle/B4g3r3qiUx6tn3RB8) que vous pouvez soumettre directement en ligne. Il existe également [une version texte](/assets/forms/fr-ph-evaluation-par-les-pairs-formulaire.txt) que vous pouvez [nous envoyer par mail](mailto:publishing.assistant@programminghistorian.org), si vous préférez. Notre édition française cherche actuellement des volontaires disponibles pour évaluer une leçon dans les 12 prochains mois.
```
Comment (marie-flesch) [inline fr/contribuer.md]: Please add the feminine form of reviewer: "Si vous souhaitez contribuer en tant qu’évaluatrice ou évaluateur, veuillez..."

Comment (anisa-hawes) [reply fr/contribuer.md]: Thank you, @marie-flesch. Done!

**Review by jenniferisasi — APPROVED** [review]
Súper!

**Review by hawc2 — APPROVED** [review]

**Comment by anisa-hawes** [comment]
Thank you @marie-flesch, @jenniferisasi, and @hawc2.

We are looking forward to supporting this initiative! I'm hopeful that it will help improve our processes for recruiting and managing peer reviewers.

**Pull request merged**